packages/tsconfig.settings.json

```diff
@@ -6,7 +6,7 @@
     "declarationMap": true,
     "sourceMap": true,
     "composite": true,
-    "lib": ["es2015", "dom"],
+    "lib": ["es2018", "dom"],
```

Comment:
Note: this lib change was required to add `padStart` to the lib. Since the compilation target was already es2018, this should not affect compatibility, just what TS flags as available.

Reply:
I see now. I was thinking why this change is necessary.

Reply:
Yes, let's not worry about old Node versions in this brand new project. Will check it out now 👍